### PR TITLE
feat: you can use a fork or a spoon for mashed potatoes

### PR DESCRIPTION
**Pull request by imnotjames**
[Feature]

## About the PR 

before people start tearing each other [apart in the forums over the right way to eat mashed potatoes](https://forum.ss13.co/showthread.php?tid=16430) I've proactively made it possible to use EITHER A FORK OR SPOON to eat mashed potatoes

This refactors all foods so the requirement for a fork or spoon is no longer exclusive

just don't use a straw for the mashed potatoes you monster

## Why's this needed? 

I eat mashed potatoes with a spoon?

## Changelog

```changelog
(u)edwardly
(+)Sure, use a spoon to eat mashed potatoes, why not?
```

### CODE REVIEW

**Comment by imnotjames** [comment]
I still need to test all of this, opened it as a draft..

**Comment by DimWhat** [comment]
Finally, the most important PR of goonstation.

**Comment by imnotjames** [comment]
dear god you can use chopsticks to eat mashed potatoes

**Comment by imnotjames** [comment]
Tested and it's working.

**Comment by nefarious6th** [comment]
A hero emerges...

File: code/modules/chemistry/tools/food_and_drink.dm [hunk 1]
```diff
@@ -191,29 +191,37 @@
...
+					if (istype(plastic_fork))
+						if (prob(20))
+							plastic_fork.break_utensil(M)
+							utensil = null
+
+					// If it's a plastic spoon we've found then test if we've broken it
+					var/obj/item/kitchen/utensil/spoon/plastic/plastic_spoon = utensil
+					if (istype(plastic_spoon))
+						if (prob(20))
+							plastic_fork.break_utensil(M)
```
Comment (Sovexe) [inline code/modules/chemistry/tools/food_and_drink.dm:212]:
```suggestion
							plastic_spoon.break_utensil(M)
```

Comment (imnotjames) [reply code/modules/chemistry/tools/food_and_drink.dm]: Ahhh, thank you!

**Pull request merged**